### PR TITLE
Port the windows inspec example

### PR DESCRIPTION
**Pull request by echohack**
Signed-off-by: echohack <echohack@users.noreply.github.com>

This adds a basic Windows example using `scaffolding-chef-inspec`

### CODE REVIEW

File: examples/windows-audit/inspec.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+maintainer: humans@habitat.sh
+copyright: Chef Software Inc.
+copyright_email: humans@habitat.sh
+license: Apache-2.0
+summary: Effortless Audit Windows Baseline
+version: 0.1.0
+
+depends:
+  - name: windows-baseline
+    url: https://github.com/dev-sec/windows-baseline/archive/1.2.0.tar.gz
```
Comment (jerryaldrichiii) [inline examples/windows-audit/inspec.yml]: Any reason to pin to this specific version over master?

Comment (echohack) [reply examples/windows-audit/inspec.yml]: Nope! I was just porting the existing version. Let's switch it

Comment (jmassardo) [reply examples/windows-audit/inspec.yml]: Could we make the version tunable?

Comment (jerryaldrichiii) [reply examples/windows-audit/inspec.yml]: If they tag and use the `git:` fetcher then yes.

Example: https://www.inspec.io/docs/reference/profiles/#git

Comment (jerryaldrichiii) [reply examples/windows-audit/inspec.yml]: Though, profiles get vendored either via Effortless or when uploading to Automate. So using latest and testing via pipeline is preferred.

Comment (echohack) [reply examples/windows-audit/inspec.yml]: I don't think I'd make that tunable for the sake of the example.

We'll do that when it comes to testing maybe?

**Pull request merged**